templates/keystoneapi/config/keystone.conf

```diff
@@ -22,7 +22,7 @@ enforce_scope = {{ .enableSecureRBAC }}
 
 [fernet_tokens]
 key_repository=/etc/keystone/fernet-keys
-max_active_keys=2
+max_active_keys={{ .fernetMaxActiveKeys }}
```

Comment:
you have to pass it into the template parameters https://github.com/openstack-k8s-operators/keystone-operator/blob/main/controllers/keystoneapi_controller.go#L1182

maybe also enhance the functional tests as part of this PR?

Reply:
I`m working on that, should I submit all together?